Sao_Paulo_brazil_sugar_content_modeling.ipynb

```diff
@@ -0,0 +1,3619 @@
+{
```

Comment:
<p>before you make a graph tell me what im going to see. let me know why it is being visualized. is this necessary? am i the user making a decision based off this graph or is this just to show our thinking along the way? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Done</p>